ontology/yaml/resources/HVAC/entity_types/SDC.yaml

```diff
@@ -71,3 +71,14 @@ SDC_EXT_TLT_WEST_SOUTHWEST:
   - SDC_EXT_TLT_WEST
   uses:
   - southwest_illuminance_sensor
+
+#Additional entity types for DE-FRA-NEM32
+SDC_EXTOR:
+  description: "Centralizend controlled directional shade with zone conditioning"
+  is_canonical: true
+  implements:
+  - SDC
+  - EXTOR
+  opt_uses:
+  - shade_orientation_specification
```

Comment:
this is metadata, indicating which direction it faces?

Reply:
Yes. It indicates the direction the belonging facade faces